torch/testing/_internal/common_methods_invocations.py

```diff
@@ -7664,6 +7684,14 @@ def reference_searchsorted(sorted_sequence, boundary, out_int32=False, right=Fal
         return np.stack(split_ret).reshape(orig_shape)
 
 
+def reference_unravel_index_without_tuple(inp, *args):
+    out_arr = np.unravel_index(inp, *args)
+    if (inp.ndim == 0):
+        return np.stack(out_arr, axis=0).reshape(-1, len(out_arr))
+    else:
+        return np.stack(out_arr, axis=1).reshape(-1, len(out_arr))
```

Comment:
This is just something I've been trying for the reference function but hasn't worked well for a few inputs so far. I'll share what exactly does it fail for in a couple of hours.

Reply:
Should be fixed now :)